CONTRIBUTING.md

```diff
@@ -1,42 +1,42 @@
-# Contributions are welcome!
+# Contributions Are Welcome!
 
-## Quick guide
+If you need any help, don't hesitate to ask the community on [Gitter](https://gitter.im/PHP-CS-Fixer/Lobby).
 
- * Fork the repo.
- * Checkout the branch you want to make changes on:
-  * Master if you make changes to the code that are not backward compatible.
-  * Default branch when adding new features.
-  * Branch before the default if you are fixing a bug for an existing feature (or the default/master branch if the feature was introduced in that version).
- * Install dependencies: `composer install`.
- * Create branch, e.g. `feature-foo` or `bugfix-bar`.
- * Make changes.
- * If you are adding functionality or fixing a bug - add a test!
- * Fix project itself: `php php-cs-fixer fix`.
- * Make sure there is no trailing spaces in code: `./check_trailing_spaces.sh`.
- * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify `README.rst` manually!
- * Check if tests pass: `phpunit` [(4.x or 5.x)](https://phpunit.de/manual/current/en/installation.html)
+## Quick Guide
 
-## Opening a pull request
+* [Fork](https://help.github.com/articles/fork-a-repo/) the repo.
+* [Checkout](https://git-scm.com/docs/git-checkout) the branch you want to make changes on:
+  * If you are fixing a bug or typo, improving tests or for any small tweak: the lowest maintained branch where the changes can be applied. Once your Pull Request is accepted, the changes will get merged up to highest branches.
```

Comment:
"Pull Request" as a link to https://help.github.com/articles/about-pull-requests/?